.github/renovate.json

```diff
@@ -312,6 +312,16 @@
       "addLabels": [
         "automerge"
       ]
+    },
+    {
+      "description": "Skip maven-surefire-plugin updates, as 3.5.3 version has a bug preventing ArchUnit tests from running. https://issues.apache.org/jira/browse/SUREFIRE-2298",
```

Comment:
❓ Do we have a way to monitor when this bug is fixed and if we can upgrade again?